src/pages/MyCoverCropList/MyCoverCropList.js

```diff
@@ -50,8 +51,32 @@ const MyCoverCropList = ({ comparisonView, from }) => {
     }
   }, [consentRedux]);
 
+  useEffect(() => {
```

Comment:
Im not sure this useEffect is the right approach to resizing the screen.  When i inspect the page i can see that the cards and sidebar are positioned in the middle of the screen as if its placed there and not placed at the top of the page.  When i add extra cards in i can see that the page will fill rather than maintaining the white space.  Can you look into how this is centering and see if you can change it to positioning from the top?

Reply:
- removed this useeffect and implemented fix at App.js level, centering only required pages